plugins/module_utils/core.py

```diff
@@ -58,6 +70,12 @@ class FlightctlModule(AnsibleModule):
             type="path",
             aliases=["ca_path"],
             fallback=(env_fallback, ["FLIGHTCTL_CA_PATH"]),
+        ),
+        flightctl_client_id=dict(
+            required=False,
+            type="str",
```

Comment:
It will also be worth to add a note or explain how the authentication works (with all the authentication options) in that doc fragment.